tests/cpp/predictor/test_gpu_predictor.cu

```diff
@@ -68,6 +66,41 @@ TEST(gpu_predictor, Test) {
   delete dmat;
 }
 
+TEST(gpu_predictor, MoreTest) {
```

Comment:
can this be combined with test which can then be invoked with varying n_cols and n_rows

Reply:
Done.